### PR TITLE
chore: open a mobile grid row as a page

### PR DESCRIPTION
**Pull request by richardshiue**
https://github.com/AppFlowy-IO/AppFlowy/assets/71320345/1c20b46e-d21a-4f29-9442-efb94e84137a

### Feature Preview

---

#### PR Checklist

- [x] My code adheres to [AppFlowy's Conventions](https://docs.appflowy.io/docs/documentation/software-contributions/conventions)
- [ ] I've listed at least one issue that this PR fixes in the description above.
- [ ] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [x] All existing tests are passing.

### CODE REVIEW

File: frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/cells/mobile_cell_container.dart [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+            ),
+          );
+        },
+      ),
+    );
+  }
+
+  BoxDecoration _makeBoxDecoration(BuildContext context, bool isFocus) {
+    if (isFocus) {
+      final borderSide = BorderSide(
```
Comment (Xazin) [inline frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/cells/mobile_cell_container.dart:71]: Why use a variable here?

File: frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/cells/mobile_cell_container.dart [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        color: Theme.of(context).colorScheme.primary,
+      );
+
+      return BoxDecoration(border: Border.fromBorderSide(borderSide));
+    }
+
+    final borderSide = BorderSide(color: Theme.of(context).dividerColor);
```
Comment (Xazin) [inline frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/cells/mobile_cell_container.dart:78]: Same here

File: frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/cells/mobile_cell_container.dart [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    );
+  }
+}
+
+class _GridCellEnterRegion extends StatelessWidget {
+  const _GridCellEnterRegion({
+    required this.child,
+    required this.accessories,
+    required this.isPrimary,
+    Key? key,
```
Comment (Xazin) [inline frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/cells/mobile_cell_container.dart:90]: Super key constructor

Comment (richardshiue) [reply frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/cells/mobile_cell_container.dart:92]: This is a private widget, so the linter will flag it

Comment (Xazin) [reply frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/cells/mobile_cell_container.dart:92]: Then remove it, if it's unused

**Review by skydiver1980 — APPROVED** [review]

**Pull request merged**